internal/controller/merge.go

```diff
@@ -213,12 +213,16 @@ func routersAreCompatible(r, toMerge *frr.RouterConfig) error {
 	return nil
 }
 
-// Verifies that two neighbors are compatible for merging, assuming they belong to the same router.
```

Comment:
why did you remove the `assuming..` part?

Reply:
I think this comment does not add value, it works toward confusing me, but no strong opinion I can add back

Reply:
I prefer we add it back, because the function really does make sense only when the neighbors belong to the same BGP router 😅